docs/docs-content/registries-and-packs/deploy-pack.md

```diff
@@ -185,34 +205,35 @@ As outlined in the [Adding Add-on Packs](adding-add-on-packs.md) guide, you can
 
 <br /> 
 
-To simplify this tutorial, we provide you with the manifest file for the *Hello Universe* application in the **packs/hello-universe-pack** folder. Change the directory to the **packs/hello-universe-pack** folder.
-<br />
+For your convenience, we provide you with the manifest-based pack files for the Hello Universe application. These files are located in the **packs/hello-universe-pack** folder. 
+
+Navigate to the **packs/hello-universe-pack** directory.
 
 ```bash
 cd /packs/hello-universe-pack
 ```
+
+
 Ensure you have the following files in the current directory.
-<br />
 
 ```bash
```

Comment:
Disable copy functionality for this code snippet perhaps? https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1947435009/Docusaurus+Migration#hide-clipboard